editors/boost-up-my-code-editor.md

```diff
@@ -0,0 +1,96 @@
+# [MO] Boost up my code editor
+
+## Owner: [Guillaume Diallo-Mulliez](https://github.com/guitoof)
+
+## ⛳️ Control Points
+
+My editor allows me to:
+
+### 🐞 **Properly Debug**: so that I can quickly identify the responsibility of an unexpected behaviour
+
+##### What ?
+
+- Setup break points
+- Inspect current context variables
+
+##### How ?
+
+- for **VSCode**: [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+
+### 🎨 **Automatically Format my Code**: so that I can follow my team syntactic standards and ease up code review and maintanability
+
+##### What ?
+
+- Real time or On save formatting
+- Inspect current context variables
+
+#### How ?
+
+- for **VSCode**: [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+
+### 🔍 **Quickly Spot Syntactic Errors**: so that I don't need to run my linter/type checker commands after each modification
+
+##### What ?
+
+- Linter
+- Type checking (flow, typescript)
+
+#### How ?
+
+- for **VSCode**:
+  - for **ESLint** : [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+  - for **TSLint** : [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint)
+
+### 📦 **Identify Outdated packages**: so that I can upgrade them rapidly after a new release
```

Comment:
I don't think this is a `must` point. It's a plus but it shouldn't be mandatory.

Reply:
This point is on of the action we took after the weekly Archi to help teams keep up with the latest versions of their packages

Reply:
@guitoof Fair point indeed but I don't think it would be your code editor job.

To check for dependencies you can do `yarn outdated` which works very nicely.

However it's still nice to check it on the fly on your editor. I'd suggest to keep this in the list, but put it below in a *recommended* section, but not mandatory 😊